json-lint/schemas/inventory/adservers/contentza.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$id": "#contentza",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Contentza adserver for TOUCH only",
+    "description": "Contentza ad server for TOUCH only",
+    "additionalProperties": false,
+    "required": [],
+    "defaultSnippets": [
+        {
+            "label": "Contentza template",
+            "description": "contains required keys",
+            "body": {}
```

Comment:
This can have the default json { id:"$1" } if you want to provide a snippet.

Reply:
sure! forget! 
I commit now , thanks m8